t/lib/t/MusicBrainz/Server/Controller/Artist/Show.pm

```diff
@@ -157,6 +157,27 @@ EOSQL
         ],
         '@context' => 'http://schema.org'
     };
+
+    $mech->get_ok('/artist/2a62773a-cdbf-44c6-a700-50f931504054');
+    page_test_jsonld $mech => {
+        'name' => 'Person A',
+        '@type' => 'MusicGroup',
```

Comment:
nitpick: since this is supposed to be a person according to the name, I assume the test data is wrong, 'cause the type isn't `['Person', 'MusicGroup']`

Reply:
It was missing types, yes. I added them now - and changed the ID for the group from 1 to 100 because 1 is hardcoded to be VA and not support a Group type! :)